### DISCUSSION TITLE
How to customize the options to show color palettes?

### DISCUSSION THREAD

**Discussion by federicomarcantognini** [janosh/svelte-multiselect#275] [Q&A]
How to customize select options with colour palettes, similar to this end result:
![image](https://github.com/janosh/svelte-multiselect/assets/19288193/720b8226-65f4-4b3b-8376-70ac71c7c50d)

I have an array of objects with label 'div ..' and label 'color palette name'
Many thanks

**Comment by janosh** [comment]
See the color scale select on this site built with `svelte-multiselect`: https://matbench-discovery.materialsproject.org/data#--chemical-diversity. [Here's the code for that](https://github.com/janosh/elementari/blob/770ffbf36e8e1aa9d68bfe2cb4f7656d11ec7fbf/src/lib/plot/ColorScaleSelect.svelte#L29).

![Screenshot 2023-11-23 at 10 13 08 AM](https://github.com/janosh/svelte-multiselect/assets/30958850/3912a604-2858-4c55-b0fb-dc62539ea296)
